### ISSUE TITLE
Certificate missing regolith-desktop.io SAN

### ISSUE THREAD

**Issue by bobdanek** [regolith-linux/regolith-desktop#664]
- [x] I've [run a search on the existing bug reports](https://help.github.com/en/github/searching-for-information-on-github/searching-issues-and-pull-requests) to make sure I'm not filing a duplicate (you can use the search box at the top left corner)
- [x] I've read [the Regolith documentation](https://regolith-linux.org/docs/) and didn't find any relevant topic
- [x] I tried [asking on Slack](https://regolith-linux.org/interact/) (especially for configuration related issues)

**Describe the bug**

Running apt updates throws errors:

```
Ign:5 https://regolith-desktop.io/release-ubuntu-jammy-amd64 jammy InRelease
Ign:5 https://regolith-desktop.io/release-ubuntu-jammy-amd64 jammy InRelease
Err:5 https://regolith-desktop.io/release-ubuntu-jammy-amd64 jammy InRelease
  Certificate verification failed: The certificate is NOT trusted. The name in the certificate does not match the expected.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]
```

```
W: Failed to fetch https://regolith-desktop.io/release-ubuntu-jammy-amd64/dists/jammy/InRelease  Certificate verification failed: The certificate is NOT trusted. The name in the certificate does not match the expected.  Could not handshake: Error in the certificate verification. [IP: 104.238.213.207 443]
```

**To Reproduce**

Try to update Regolith packages via apt

**Expected behavior**

Updates and upgrades succeed without error

**Screenshots**

Not applicable, only terminal output as shared above

**Configuration file(s)**

Not applicable (I think/hope, happy to dive into this if needed though).

**Installation Details**
 - Regolith Install Type: PPA
 - Regolith Version: `regolith-desktop/unknown,now 4.1.0-1-1regolith amd64`
 - PPA url: https://regolith-desktop.io/release-ubuntu-jammy-amd64

 - Host OS (for PPA): Pop_OS! 22.04 LTS

**Additional context**
Certificate common name is regolith-desktop.org, and there's no subject alternate name for regolith-desktop.io:

```Shellsession
$ openssl s_client -servername regolith-desktop.io -connect regolith-desktop.io:443 2>/dev/null | openssl x509 -nooout -text | grep regolith
        Subject: CN = regolith-desktop.org
                DNS:regolith-desktop.org
```

Also mentioned in Slack: https://regolith-linux.slack.com/archives/CLM0Y3H5Y/p1658254349489399

**Comment by kgilmer** [comment]
Thanks for reporting the issue.  The root cause of this is that a domain registered w netlify was switched to another host but the certificate seems to still be registered against the original host.  Unclear to me at this time how to fix this, may need to contact netlify.

**Comment by bobdanek** [comment]
> Thanks for reporting the issue. The root cause of this is that a domain registered w netlify was switched to another host but the certificate seems to still be registered against the original host. Unclear to me at this time how to fix this, may need to contact netlify.

It's a Let's Encrypt certificate, and it may be configurable by the site maintainers as described under "Getting Started" here: https://www.netlify.com/blog/2016/01/15/free-ssl-on-custom-domains/

Just a blind guess tho, not sure how the site is set up or anything. Hope that's helpful!

**Comment by kgilmer** [comment]
A temporary fix is to update the URL to Http instead of https

**Comment by grhoades** [comment]
or temporarily use the .org extension everywhere you see the .io extension in the instructions for install?

**Comment by ferdinandyb** [comment]
I think it is related, but I'm having trouble importing the gpg key as well.

**Comment by grhoades** [comment]
Yes, same issue with the key wget request and the domain suffix (.io vs. org)

wget -qO - https://regolith-desktop.io/regolith.key | \
gpg --dearmor | sudo tee /usr/share/keyrings/regolith-archive-keyring.gpg > /dev/null

**Comment by kgilmer** [comment]
Still waiting on a response from Netlify (they issued the cert)

**Comment by bobdanek** [comment]
Changing my source to .org instead of .io, going through an apt update/upgrade (which failed with GPG error), then re-adding the keyring got me in good shape again.

Regarding the certificate, another approach could be to simply issue a new cert for the .io domain and have Netlify automatically manage both of them separately. That's likely more straightforward anyways.

I would expect the account holder to be able to address this directly instead of needing action from the web host, but of course I'm only making blind guesses having never used Netlify before.

Anyways, if they don't get back to you in a reasonable amount of time, these might be helpful:
- https://docs.netlify.com/domains-https/https-ssl/#netlify-managed-certificates
- https://docs.netlify.com/domains-https/troubleshooting-tips/#certificates-and-https

**Comment by kgilmer** [comment]
The service provided that issued the cert has not responded to the ticket, breaching SLA.  I have opted to switch the instructions on the website to use the ".org" version, of which I control the cert because I issued it myself.  Going forward we'll continue with regolith-desktop.org for the debian repo and at some point when the .io domain is under control we'll do something else with that.  Sorry for the confusion.

Thanks for the tips @bobdanek , this sort of thing is not my forte and while I did read some docs regarding how I may revoke a cert issued on my behalf, I didn't feel the risk was acceptably low in impacting the DNS issue further.

Kindly re-open this ticket if anyone finds problems with the updated instructions on the website.
